tests/services/repo-creator.service.spec.ts

```diff
@@ -37,6 +41,33 @@ describe('RepoCreatorService', () => {
     });
   });
 
+  describe('.requestRepoCreationPermissions()', () => {
+    it('should not return any permissions if the repo was already created', () => {
+      of(true)
+        .pipe(repoCreatorService.requestRepoCreationPermissions(Phase.phaseBugReporting, PHASE_REPO))
+        .subscribe((repoCreationPermission: boolean | null) => expect(repoCreationPermission).toBe(null));
+    });
+
+    it('should not return any permissions if the repo was not created and the current phase is not bugReporting', () => {
+      of(false)
+        .pipe(repoCreatorService.requestRepoCreationPermissions(Phase.phaseModeration, PHASE_REPO))
+        .subscribe((repoCreationPermission: boolean | null) => expect(repoCreationPermission).toBe(null));
+    });
+
+    it('should return permissions from matDialog if the repo was not created and current phase is bugReporting', () => {
+      userService.currentUser = USER_JUNWEI;
+      matDialog.open.and.callFake(() => matDialogRef);
+      const permissionFromDialog = false;
+      matDialogRef.afterClosed.and.callFake(() => of(permissionFromDialog));
+      of(permissionFromDialog)
```

Comment:
```suggestion
      of(false)
```
Think you made a typo here.
The boolean here isn't the permission from the dialog. It indicates whether the repo already exists / does not exist.

Reply:
:+1: